SpamPkg/Library/SmmCpuFeaturesLib/SmmStm.c

```diff
@@ -329,6 +340,7 @@ DiscoverSmiEntryInFvHobs (
         }
       } while (!EFI_ERROR (Status));
       Hob.Raw = GetNextHob (EFI_HOB_TYPE_FV, GET_NEXT_HOB (Hob));
+      Status = EFI_SUCCESS;

```

Comment:
Do you want to add this right before the `Done:` symbol?

Reply:
Moved this line to be in the passing condition (found all three binaries).